common/hooks/useIsFontsLoaded.ts

```diff
@@ -15,9 +16,14 @@ const useIsFontsLoaded = (): boolean => {
 
     Promise.all([WB.load(), HNR.load(), HNB.load(), LR.load()])
       .then(() => {
-        setIsFontsLoaded(true);
+        if (isMounted) {
+          setIsFontsLoaded(true);
+        }
       })
       .catch(console.log);
+    return () => {
+      isMounted = false;
+    };
```

Comment:
Could you explain what is happening here?

Reply:
We can't cancel promises, so using the isMounted value to prevent the component from trying to update the state if it's been unmounted.

Reply:
Might sound like a broken record, but personally a comment would help me remember this in the future.